### ISSUE TITLE
HackStudio: Navigating to folder with pictures causes eternal hangup

### ISSUE THREAD

**Issue by maxgerhardt** [SerenityOS/serenity#7127]
Serenity OS commit: 3ba3d2ddfdb44e28323b4f6f5a1d7f98d8b4be23
Running on: QEMU, Ubuntu 20, x64

I've observed the behavior that when I
* mount the `_disk_image` locally to some directory
* clone in some repository of mine that has pictures in it (https://github.com/gamer-cndg/omega2-gnuboy)
* unmount the disk
* start Serenity
* start HackStudio 
* use "Open Project"
* navigate to the cloned folder

The "Open project" dialog will hang up upon attempting to open the folder. The window becomes unresponsive, even after a long time of waiting. After some time, the entire SerenityOS freezes up completely (no mouse movement possible anymore) and the only possible thing is to close QEMU and restart.

![grafik](https://user-images.githubusercontent.com/26485477/118352542-4ee53a00-b562-11eb-9856-bc404df81cee.png)

After restarting SerenityOS, I can still open the aforementioned folder in the File Manager, in which it slowly generates thumbnails. After then deleting the pictures in the folder, HackStudio can open the folder no problems and does not hang up anymore. Thus I think the pictures / thumbnails are part of the problem here.

**Comment by bcoles** [comment]
Do you get any interesting/useful output in the debug console?

**Comment by maxgerhardt** [comment]
I did have the system monitor open but it did not show significant CPU usage,, I think the WIndowServer was the highest with like 20%. Also no error messages were in the QEMU log.

However, I just retried this after learning I actually have to recompile the SerenityOS kernel with `ninja install` when I pull updates, and after that I tried the same procedure per above and I cannot reproduce my hangup anymore... Maybe it was already fixed. 

Closing as seemingly invalid now.
